mlblocks_primitives/sklearn.neighbors.KNeighborsRegressor.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "sklearn.neighbors.KNeighborsRegressor",
+    "author": "Sze Nga Wong (wsnalice@mit.edu)",
+    "documentation": "http://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsRegressor.html",
+    "description": "Regression based on k-nearest neighbors. The target is predicted by local interpolation of the targets associated of the nearest neighbors in the training set."
```

Comment:
This line is missing a comma at the end, which makes the JSON syntax invalid. Can you add it?